modules/web/src/app/shared/components/application-list/edit-application-dialog/component.ts

```diff
@@ -158,7 +154,7 @@ export class EditApplicationDialogComponent implements OnInit, OnDestroy {
         this._builder.control(
           {
             value: this.application.namespace,
-            disabled: this.isEnforcedApplication(),
+            disabled: this.isEnforcedApplication() ? true : isSystemApplication(this.application?.labels),
```

Comment:
This will also prevent users from editing the values on cluster details page.

Reply:
do we want them to edit these values ? in the cluster details page? cause it's system applications ?

Reply:
Yes, we do. They can also edit Cilium application values.

Reply:
where ?
<img width="653" height="691" alt="image" src="https://github.com/user-attachments/assets/ba4d33c2-45f0-4668-8c06-5d66e536275a" />

Reply:
Sorry @ahmadhamzh you're correct. I confused this with something else.